test/parallel/test-memory-usage.js

```diff
@@ -24,6 +24,15 @@
 const common = require('../common');
 const assert = require('assert');
 
+const result = process.memoryUsage({ rssOnly: true });
```

Comment:
```suggestion
process.memoryUsage();
const result = process.memoryUsage({ rssOnly: true });
```
Then re-run the test. :-)

(You need to clear the other fields when rssOnly=true or they'll contain stale values.)